### PR TITLE
[Java] fix library version information

### PR DESCRIPTION
**Pull request by cecile75**
## Motivation

Getting the correct library version information so we can use context.library in the @missing_feature annotation.

**Before:**

- context.library always contains the value of the latest java tracer

**Now:**

- If there is no java agent in /binaries., context.library contains the version of the latest java tracer
- If there is a java agent in /binaries, context.library contains its version.

Remark: the new libraryversion.sh file has some redundancy with run.sh but at least it's working.

## Changes

Add a small bash to fill out /client/SYSTEM_TESTS_LIBRARY_VERSION in both cases: custom java agent or not.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: utils/build/docker/java/parametric/libraryversion.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (cbeauchesne) [inline utils/build/docker/java/parametric/libraryversion.sh:1]: There is a small convention, could you rename this file to `install_ddtrace.sh` ?

Comment (cecile75) [reply utils/build/docker/java/parametric/libraryversion.sh:1]: @cbeauchesne Even is the selection of the tracer to run is done in run.sh? 
libraryversion.sh only stores the tracer version in SYSTEM_TESTS_LIBRARY_VERSION

**Review by cbeauchesne — COMMENTED** [review]
Co-reviwed by zoom, waiting for the CI

File: utils/build/docker/java/parametric/libraryversion.sh (hunk 1 above)
Comment (cecile75) [reply utils/build/docker/java/parametric/libraryversion.sh]: This was written during the zoom and install_ddtrace.sh was deleted

**Review by cbeauchesne — APPROVED** [review]
Issue are not related, you can merge

**Pull request merged**